project_event/views/project_task_view.xml

```diff
@@ -320,4 +321,28 @@
               parent="project_event_menu"
               action="action_project_event_task"
               sequence="30"/>
+    
+    <record id="calendar_color_project_task_color_view" model="ir.ui.view">
```

Comment:
@eilst Why you didn't put this code in the same view : project_event_task_calendar